tests/suites/test_suite_x509parse.data

```diff
@@ -1670,3 +1670,18 @@ X509 Get time (UTC invalid character in sec)
 depends_on:MBEDTLS_X509_USE_C
 x509_get_time:MBEDTLS_ASN1_UTC_TIME:"0011302359n0Z":MBEDTLS_ERR_X509_INVALID_DATE:0:0:0:0:0:0
 
+X509 Get time (Generalized Time invalid leap year multiple of 4 and 100)
```

Comment:
The description is wrong (the property of the year is “100 but not 400”, not “4 and 100”, and it isn't an “invalid leap year”). Suggestion:

> X509 Get time (Generalized Time, year multiple of 100 but not 400 is not a leap year)